### ISSUE TITLE
Choose the init state

### ISSUE THREAD

**Issue by fallais** [ziscloud/angular-bootstrap-toggle#29]
Is it possible to choose the init state please ? Directly in the view I mean, not in the controller (which is what I actually do)

**Comment by dk8996** [comment]
Yes this is a big bug.  When creating a variable via ng-init and using it in ng-model the toggle doesn't work.
